tests/binary.rs

```diff
@@ -30,14 +30,14 @@ mod binary {
   #[cfg(not(windows))]
   fn get_rav1e_command() -> Command {
     let mut cmd = Command::cargo_bin("rav1e").unwrap();
-    cmd.env_clear();
+    cmd.arg("-y").env_clear();
```

Comment:
I think it makes more sense to put the arg under `get_common_cmd`

Reply:
Sure, and done :)